voucherApp/src/test/java/com/mountain/voucherApp/service/DefaultVoucherAppServiceTest.java

```diff
@@ -1,13 +1,14 @@
-package com.mountain.voucherApp.application.service;
-
-import com.mountain.voucherApp.adapter.out.persistence.voucher.VoucherEntity;
-import com.mountain.voucherApp.application.port.in.CustomerDto;
-import com.mountain.voucherApp.application.port.in.VoucherCreateDto;
-import com.mountain.voucherApp.application.port.in.VoucherIdUpdateDto;
-import com.mountain.voucherApp.application.port.out.CustomerPort;
-import com.mountain.voucherApp.application.port.out.VoucherPort;
-import com.mountain.voucherApp.domain.vo.CustomerName;
-import org.hamcrest.core.IsNull;
+package com.mountain.voucherApp.service;
```

Comment:
패키지명은 카멜케이스 스네이크 케이스 사용하고 있어요. 
```suggestion
package com.mountain.voucher-app.service;
```

Reply:
아래 링크 참조하여 스네이크 케이스아닌 모두 소문자인 `voucherapp` 으로 변경해보았습니다 !
- [참조](https://stackoverflow.com/questions/3179216/what-is-the-convention-for-word-separator-in-java-package-names)